crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -1320,6 +1413,27 @@ pub fn validate_conditions(
         }
     }
 
+    if !state.messages.is_empty() {
+        // the integers count the number of times the message has been sent
+        // minus the number of times it's been received. At the end we ensure
+        // all counters are 0, otherwise some message wasn't received or sent
+        // the right number of times.
+        let mut messages = HashMap::<Vec<u8>, i32>::new();
+
+        for msg in &state.messages {
+            *messages.entry(msg.make_key(a)).or_insert(0) += msg.counter as i32;
```

Comment:
According to docs, this is recommended to be something like `messages.entry(msg.make_key(a)).and_modify(|c| *c += msg.counter as i32).or_insert(msg.counter as i32);`

Reply:
I'm not convinced that's an improvement in readability